src/pkg/fault/alert.go

```diff
@@ -0,0 +1,117 @@
+package fault
```

Comment:
Is this purely a code movement from earlier PR in the chain? Could this change be done in the earlier PR itself?

Reply:
It is.  Aplogies, I'll try to separate the two in future PRs.